### PR TITLE
Update to match latest IdeaVim API

### PR DESCRIPTION
**Pull request by lippfi**
https://github.com/TheBlob42/idea-which-key/issues/56
(It doesn't close the issue, though. There might be other compatibility issues in the future)

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -33,7 +33,7 @@ tasks.withType<PatchPluginXmlTask> {
 intellij {
     version.set("2023.3.2")
     updateSinceUntilBuild.set(false)
-    plugins.set(listOf("IdeaVIM:2.8.0"))
+    plugins.set(listOf("IdeaVIM:2.10.0"))
```
Comment (TheBlob42) [inline build.gradle.kts:36]: I get the following error when trying to build this locally:
```
Cannot resolve plugin 'IdeaVIM' in version '2.10.0'
```
Is this missing corresponding the "EAP" postfix or did you add this here for when `2.10.0` becomes available and you don't want this PR to be merged beforehand anyway?

Comment (lippfi) [reply build.gradle.kts:36]: Apologies for the lack of description. Yes, this PR is intended for the future, when version 2.10 is released.

Comment (TheBlob42) [reply build.gradle.kts:36]: Thanks for the explanation and your efforts :bow:

**Comment by Inmoresentum** [comment]
Its time to merge it. Because which key now shows normal mode keybindings when typing in insert mode which is kind of annoying.

**Pull request merged**